acceptance/indy_acceptance/indy-tests/utils/constant.py

```diff
@@ -0,0 +1,32 @@
+'''
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+'''
+
+
+class Colors:
+    """ Class to set the colors for text.  Syntax:  print(Colors.OKGREEN +"TEXT HERE" +Colors.ENDC) """
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'  # Normal default color
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class Roles:
+    TRUSTEE = "TRUSTEE"
+    STEWARD = "STEWARD"
+    TRUST_ANCHOR = "TRUST_ANCHOR"
+    TGB = "TGB"
+    NONE = ""
+
+
+class Constant:
+    import os
+    work_dir = os.path.expanduser('~') + os.sep + ".indy"
+    seed_default_trustee = "000000000000000000000000Trustee1"
+    pool_genesis_txn_file = os.path.expanduser('~') + os.sep + ".sovrin/pool_transactions_sandbox_genesis"
```

Comment:
In the latest master and RC of indy-node, pool_genesis_txn_files are located in 
`/var/lib/indy/sandbox/pool_transactions_genesis` (for sandbox network)

Reply:
This path is configurable so the user can use multiple transaction files in multiple locations

Reply:
yes, but if you take config files from a sovrin installation, then they will be placed to the location mentioned above (and not to .sovrin)

Reply:
Khoi, please add a comment above this line that tells the user the path to the genesis transaction file is configurable.  The comment should indicate the default directory for the genesis transaction file is /var/lib/indy/sandbox/ but the user can put the genesis transaction file anywhere they want as long as the path is set in this line.  Please remove .sovrin/ and put the default path as /var/lib/indy/sandbox/ before checking this file in.